README.md

```diff
@@ -49,6 +49,11 @@ as [Wikimedia Germany](https://wikimedia.de) employee for the [Wikidata project]
 
 ## Release notes
 
+### 1.1.0 (2016-05-24)
+
+* `DataValueDeserializer` optionally accepts callbacks, in addition to full qualified DataValue
+  class names and assuming they implement a static `newFromArray` method.
```

Comment:
I suggest this wording instead: https://github.com/DataValues/Serialization/pull/17